### DISCUSSION TITLE
Response for new Marketing Chanels

### DISCUSSION THREAD

**Discussion by andrelsouza** [facebookexperimental/Robyn#767] [Q&A]
Hello, guys.

I'm studying this excelent Robyn package and for specifics Marketing channels that we adopted recently, our Budget Allocator suggests that the % spent is 0, if my understanding is right, this is because of the low historical data of these channels, does it make sense? If so, how Robyn process and deal with these new channels?

**Comment by gufengzhou** [comment]
Sorry for the late reply. New channel means sparse data, which is a challenge for any models. "small channel" usually means small spend, which doesn't always mean small effect. Regression is correlation based. So a small but well correlating channel could also get larger contribution. It depends.
